### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by vehagn**
# Pull Request

## What? (description)

My attempt at clarifying the readme (solving #192).
The edits are up for discussions of course!

## Why? (reasoning)

The readme was somewhat difficult to follow, I've tried to clean it up a bit.

## Acceptance

Please use the following checklist:

- [x] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [ ] ~~you linted your code (`make lint`)~~ (missing .golangci.yml file)
- [x] you linted your code (`make unit`)

> See `make help` for a description of the available targets.

### CODE REVIEW

**Review by sergelogvinov — COMMENTED** [review]
Thank you for the contribution! Can you merge 2 commits in one, and we can merge it

Thanks.

File: README.md [hunk 1]
```diff
@@ -92,23 +194,9 @@ Storage parameters:
...
-```
-
-Create user and grant permissions:
-
-```shell
-pveum user add kubernetes-csi@pve
-pveum aclmod / -user kubernetes-csi@pve -role CSI
-pveum user token add kubernetes-csi@pve csi -privsep 0
-```
-
```
[the remark this answers is not in the document]
Comment (vehagn) [reply README.md:110]: This should absolutely be kept! I moved it up together with the clustering of Proxmox.

**Review by sergelogvinov — APPROVED** [review]
Thank you!

File: docs/pvecsictl.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ Available Commands:
 
 ### Migrate
 
-Migration requires the root privileges on the Proxmox cluster.
-You need to provide the cloud-config file with the root credentials (username/password) to the Proxmox cluster.
+Migration requires root privileges on the Proxmox cluster.
```
Comment (vehagn) [inline docs/pvecsictl.md:24]: @sergelogvinov I assume it's possible to create a role for this similar to the one for the `kubernetes-csi@pve` user?

Comment (sergelogvinov) [reply docs/pvecsictl.md:24]: Unfortunately not... 
This is proxmox limitation. https://pve.proxmox.com/pve-docs/api-viewer/#/nodes/{node}/storage/{storage}/content/{volume} (POST method)

Comment (vehagn) [reply docs/pvecsictl.md:24]: Thanks for the clarification. I'll add that to the documentation.

**Pull request merged**